fennel/datasets/datasets.py

```diff
@@ -2394,10 +2430,14 @@ def visitAggregate(self, obj) -> DSSchema:
                     )
             else:
                 raise TypeError(f"Unknown aggregate type {type(agg)}")
+
+        timestamp_field = (
+            obj.along if obj.along is not None else input_schema.timestamp
+        )
         return DSSchema(
             keys=keys,
             values=values,  # type: ignore
-            timestamp=input_schema.timestamp,
+            timestamp=timestamp_field,
```

Comment:
Does this mean that the name of the along field becomes the name of the timestamp field in output dsschema? If so, that's not the desired behavior. The timestamp field of the input dataset should still be the name of the timestamp field in the output dataset - just that lookup will behave as if it was along a different axis.